free-programming-books-id.md

```diff
@@ -84,6 +87,11 @@
 * [Tutorial Dasar Javascript untuk Pemula](https://www.petanikode.com/tutorial/javascript/) *(Dalam Proses)*
 
 
+### Misc
+
+* [Kumplan Ebook Dari Berbagai Sumber](https://gilacoding.com/ebook) - Gila Coding
```

Comment:
It's a list of books taken form other sites, including books already in our list.